media-api/app/lib/querysyntax/Parser.scala

```diff
@@ -2,8 +2,10 @@ package lib.querysyntax
 
 object Parser {
 
-  def run(input: String): List[Condition] =
-    normalise(parse(input))
+  def run(input: String): List[Condition] = {
+    val inputString = if(!input.contains("is:deleted")) input.concat(" is:not-deleted").trim else input
+    normalise(parse(inputString))
```

Comment:
I think could be expressed more clearly as...
```suggestion
    normalise(
      parse(
        if(input.contains("is:deleted")) input
        else input.concat(" is:not-deleted").trim 
      )
    )
```

Reply:
Also, did you try @paperboyo 's suggestion of `-is:deleted` (where the `-` makes it a negation of `is:deleted`) (see https://github.com/guardian/grid/pull/3376#issuecomment-875768691)? 

_Happy to stick with `not-deleted` btw, as its slightly more explicit/readable unless there is no precedent amongst other filters?_

Reply:
@twrichards yes i tried `-is: deleted`, i will update the PR with it now, it makes sense, rather than introducing a negation filter

Reply:
If the default is to hide deleted images, then this js change shouldn't be needed and we should just have an "is not deleted" filter running by default that `is:deleted` reverses. 

I think it may also make sense to have a filter which shows both deleted and not images.

Reply:
> I think it may also make sense to have a filter which shows both deleted and not images.

Was thinking about that too. But the problem is: Grid has no `OR` searches **at all** (which is bad, very bad).

Reply:
If the default state is to search for images which are not deleted, then this should not be a seperate filter. We should use the presence of the `deleted` flag to change the `deleted` filter from showing not-deleted images to showing deleted images. I'd also suggest a filter which shows both deleted and not-deleted images. I believe one of the free to use filters already works like this.

Reply:
I think it is a similar approach here
https://github.com/guardian/grid/blob/cae0b648ef0804cbfc985cc52d9fd5d86d4adeeb/media-api/app/lib/querysyntax/Parser.scala#L8